tunnel/README.md

```diff
@@ -0,0 +1,52 @@
+# Tunnel
+
+## Overview
+
+This example illustrates how to connect application runnig on local machine to Kyma cluster.
+
+## Prerequisites
+
+- [Kyma](https://kyma-project.io/docs/) as the target deployment environment
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) configured to your Kyma cluster
+- [Helm](https://helm.sh/docs/intro/install/)
```

Comment:
You might want to indicate that the required version of Helm is v3.X.X, with previous versions the latter commands do not work properly (arguments mismatch).

Reply:
Fixed.